$NetBSD: patch-roff.7,v 1.2 2013/10/05 14:48:15 wiz Exp $

Install preconv and roff with 'mandoc-' prefix to avoid conflict with groff.

--- roff.7.orig	2013-09-29 23:34:34.000000000 +0000
+++ roff.7
@@ -16,10 +16,10 @@
 .\" OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\"
 .Dd $Mdocdate: August 8 2013 $
-.Dt ROFF 7
+.Dt MANDOC-ROFF 7
 .Os
 .Sh NAME
-.Nm roff
+.Nm mandoc-roff
 .Nd roff language reference for mandoc
 .Sh DESCRIPTION
 The
